content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -134,11 +133,11 @@ Edit the config file with a text editor of your choice, such as Notepad.
 
 ### Enable shell autocompletion
 
-kubectl provides autocompletion support for Bash and Zsh, which can save you a lot of typing.
+kubectl provides autocompletion support for Bash, Zsh, Fish, and Powershell, which can save you a lot of typing.
 
-Below are the procedures to set up autocompletion for Zsh, if you are running that on Windows.
+Below are the procedures to set up autocompletion for Powershell.
 
-{{< include "included/optional-kubectl-configs-zsh.md" >}}
+{{< include "included/optional-kubectl-configs-pwsh.md" >}}
```

Comment:
are you removing the inline zsh on windows instructions? am I understanding this correctly?

Reply:
Yes, primarily because I don't think Windows zsh is a common installation. Especially since the Windows port of zsh is very old: http://zsh-nt.sourceforge.net/

I'd imagine for the WSL installation, people should look at the linux installation guide.

Reply:
Thanks for clarifying. I agree with this change.